src/services/v2/utils/errorHandler.js

```diff
@@ -0,0 +1,51 @@
+const ERROR_MESSAGES = {
+  CONNECTION_ERROR: 'Unable to connect to the server. Please check your internet connection.',
+  UNEXPECTED_ERROR: 'An unexpected error occurred. Please try again.'
+}
+
+const formatPath = (path) => {
+  const prefix = '/data/'
+
+  if (!path.startsWith(prefix)) return null
+
+  const rest = path.slice(prefix.length)
+  if (!rest) return null
+
+  return rest
+    .split('/')
+    .map((part) => part.charAt(0).toUpperCase() + part.slice(1))
+    .join(' - ')
+}
+
+/**
+ * Handles API errors and returns a standardized error response
+ * @param {Object} error - The error object from the API call
+ * @returns {Object} Standardized error response
+ */
+export function errorHandler(error) {
+  if (!error?.response) {
+    return {
+      status: 500,
```

Comment:
Would it be interesting to add case for other standard errors, such as 404 and 403?